manage.py

```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 import venv
+import logging
+
+#logging root configuration
+logging.basicConfig(level=logging.INFO, # This will log all levels of messages
```

Comment:
One more comment here. This setting will  avoid logging any debug messages, so it does not log all levels of messages.

Reply:
I think when need to debug,  set the debug level to DEBUG and return it to default after end debugging.
something like this
`    if debug_mode:
        previous_level = logging.getLogger().level  # Store current level
        logging.getLogger().setLevel(logging.DEBUG)  # Set to DEBUG for this block
        logging.debug(debug_message)
        logging.getLogger().setLevel(previous_level)`

Reply:
I will have variable debug_mode which it's default is false but when it's true i will change the Level to Debuger.